rebar.config

```diff
@@ -200,7 +200,46 @@
                                         "{{output_dir}}/bin/check_config"},
                                        {copy,
                                        "_build/dev1/lib/aeutils/priv/aeternity_config_schema.json",
-                                       "{{output_dir}}/data/aeternity_config_schema.json"}]}
+                                       "{{output_dir}}/data/aeternity_config_schema.json"},
+                                       {copy,
+                                       "config/dev1/keys/{peer_key,peer_key.pub}",
+                                       "{{output_dir}}/data/aecore/keys/"}]}
+                           ]},
+                     {erl_opts, [{d, 'TEST'}]}
+                    ]
+            },
+            {dev2, [{relx, [{dev_mode, false},
+                            {include_erts, false},
+                            {sys_config, "./config/dev2/sys.config"},
+                            {vm_args, "./config/dev2/vm.args"},
+                            {overlay, [{mkdir, "{{output_dir}}/data"},
+                                       {copy,
+                                        "_build/dev2/bin/check_config",
+                                        "{{output_dir}}/bin/check_config"},
+                                       {copy,
+                                       "_build/dev2/lib/aeutils/priv/aeternity_config_schema.json",
+                                       "{{output_dir}}/data/aeternity_config_schema.json"},
+                                       {copy,
+                                       "config/dev2/keys/{peer_key,peer_key.pub}",
```

Comment:
relx supports this copy format `{X,Y,Z}`, might be worth using in other copy sections above and reduce the verbosity